app/views/passengers/index.haml

```diff
@@ -14,6 +14,10 @@
               checked: @filter == filter,
               inline: true
       = f.submit 'Print'
+    -if @current_user.admin?
+      %br
```

Comment:
No `<br>` tags, please.

Note that I kinda prefer the button to the right rather than below. I just don't know if it should be within the `form` DOM-wise.

Reply:
I can't seem to figure out a way to do that. I might need some help.